worlds/dark_souls_3/docs/locations_en.md

```diff
@@ -922,12 +922,12 @@ offline _Dark Souls III_ randomizer].
 <tr><td>CD: Armor of Thorns - Rosaria&#x27;s Bed Chamber after killing Kirk</td><td>Found in Rosaria&#x27;s Bed Chamber after killing Longfinger Kirk</td></tr>
 <tr><td>CD: Astora Greatsword - graveyard, left of entrance</td><td>Down one of the side paths to the left in the Reanimated Corpse area</td></tr>
 <tr><td>CD: Barbed Straight Sword - Kirk drop</td><td>Dropped by Longfinger Kirk when he invades in the cathedral central room</td></tr>
-<tr><td>CD: Black Eye Orb - Rosaria from Leonhard&#x27;s quest</td><td>On Rosaria&#x27;s corpse after joining Rosaria&#x27;s Fingers, exhausing Leonhard&#x27;s dialogue there and reaching the Profaned Capital bonfire.</td></tr>
+<tr><td>CD: Black Eye Orb - Rosaria from Leonhard&#x27;s quest</td><td>On Rosaria&#x27;s corpse after joining Rosaria&#x27;s Fingers, exhausting Leonhard&#x27;s dialogue there and reaching the Profaned Capital bonfire.</td></tr>
```

Comment:
We shouldn't fix issues in the descriptions from the static randomizer unless we're also fixing them upstream (https://github.com/thefifthmatt/SoulsRandomizers/blob/master/dist/Base/annotations.txt)

Reply:
So should these typoes all just be left in? That seems very strange to me: "exhausing" "flamerge" "overloop" "cemetary" "balista" "Crufixion" "minic" "rafers" "acessed" "rooftopps" "Pestlient" "entrace" "pavillion" "an left'

Reply:
I opened a PR to the upstream with the same changes now